lib/registries/addon/overview/controller.ts

```diff
@@ -38,6 +38,25 @@ export default class Overview extends Controller {
         return 'drawer';
     }
 
+    @computed('registration.{embargoed,embargoEndDate}')
+    get embargoEndDate() {
+        if (!this.registration) {
+            return;
+        }
+
+        if (this.registration.embargoed && this.registration.embargoEndDate) {
+            const [
+                weekDay,
+                month,
+                day,
+                year,
+            ] = this.registration.embargoEndDate.toDateString().split(' ');
+            return `${weekDay}, ${month} ${day}, ${year}`;
```

Comment:
Probably better to use [`{{moment-format}}`](https://github.com/stefanpenner/ember-moment#moment-format) in the template for now (until we upgrade from `ember-i18n` to `ember-intl`).

Also, per [this ticket](https://openscience.atlassian.net/browse/EMB-249) we're trying to use the same date format (in moment syntax: `MMMM DD, YYYY`) consistently.